### ISSUE TITLE
Mark event as finished

### ISSUE THREAD

**Issue by devnoname120** [leits/MeetingBar#469]
Sometimes a meeting ends earlier than expected.
I would like to be able to skip it and see when the next event is happening.

Changing my calendar entry is too bothersome and MeetingBar takes some minutes to pick it up anyway.

**Comment by adamseadub** [comment]
I actually came to the repo to see if this had been discussed. I'd also like to be able to dismiss a meeting. For an example that's happening right now: I just finished a call early, and have another meeting in about 45 minutes. 
MeetingBar still shows the existing  ended call. If I could dismiss it, it would show the upcoming event which is currently the more relevant info.

Just a side note: Thanks to all the devs and contributors to this project. It's been incredibly helpful. I wish I had the dev hours to spare to make this feature request via PR, but alas I can't at the moment.
